spec/facebook_post_generator_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'cute_pets/post_generator'
+require 'minitest/autorun'
+
+describe 'FacebookPostGenerator' do
+  describe '.create_message' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.